ci/builders/mac_ios_engine.json

```diff
@@ -294,6 +294,61 @@
                 }
             }
         },
+        {
```

Comment:
This should go in mac_unopt.json

Reply:
Actually, there are already builders with similar targets defined in there https://github.com/flutter/engine/blob/d13cbfb6f38afb5f974a1f4a784c9e0e06dd6ac8/ci/builders/mac_unopt.json#L497, and we have never seen them fail before.
All of the failures we have seen were on mac_ios_engine.